hc/front/tests/test_add_pushover.py

```diff
@@ -0,0 +1,46 @@
+from django.test.utils import override_settings
+from hc.api.models import Channel
+from hc.test import BaseTestCase
+
+
+@override_settings(PUSHOVER_API_TOKEN="token", PUSHOVER_SUBSCRIPTION_URL="url")
+class AddPushoverTestCase(BaseTestCase):
+    def test_it_adds_channel(self):
+        self.client.login(username="alice@example.org", password="password")
+
+        session = self.client.session
+        session["po_nonce"] = "n"
+        session.save()
+
+        params = "pushover_user_key=a&nonce=n&prio=0"
+        r = self.client.get("/integrations/add_pushover/?%s" % params)
+        assert r.status_code == 302
+
+        channels = list(Channel.objects.all())
+        assert len(channels) == 1
```

Comment:
Pytest uses just plain `assert` statements.. but `assert` statements are not limited to pytests only. You can use them within your code as needed.

AFAIK, there isn't any particular advantage of using one over another. However, it is important to stick to a consistent style within scripts.

In our case, we are using self.assertStuff() for all tests (and 👎 , it is not cool to mix it up).